rust/operator-binary/src/config.rs

```diff
@@ -70,9 +76,19 @@ pub fn build_bootstrap_conf(overrides: BTreeMap<String, String>) -> String {
         "java.arg.1".to_string(),
         "-Dorg.apache.jasper.compiler.disablejsr199=true".to_string(),
     );
-    // JVM memory settings
-    bootstrap.insert("java.arg.2".to_string(), "-Xms1024m".to_string());
-    bootstrap.insert("java.arg.3".to_string(), "-Xmx1024m".to_string());
+
+    // TODO: log failure to compute Java heap size
```

Comment:
Probably don't want to fail silently here

Reply:
Fixed now.